src/PostgREST/Error.hs

```diff
@@ -530,8 +530,7 @@ instance JSON.ToJSON Error where
   toJSON (SingularityError n) = JSON.object [
     "code"    .= ApiRequestErrorCode16,
     "message" .= ("JSON object requested, multiple (or no) rows returned" :: Text),
-    "details" .= T.unwords ["Results contain", show n, "rows,", T.decodeUtf8 (MediaType.toMime MTSingularJSON), "requires 1 row"],
-    "hint"    .= JSON.Null]
```

Comment:
For consistency, we should keep the `hint` as null. Check a previous discussion about this here: https://github.com/PostgREST/postgrest/issues/2176#issuecomment-1049982360

Once that's corrected, it'll be good to merge.